sdk/src/Core/Amazon.Runtime/Credentials/AnonymousIdentityResolver.cs

```diff
@@ -26,20 +26,22 @@ public class AnonymousIdentityResolver : IIdentityResolver<AnonymousAWSCredentia
     {
         private readonly AnonymousAWSCredentials _credentials = new();
 
-        BaseIdentity IIdentityResolver.ResolveIdentity() => _credentials;
+        BaseIdentity IIdentityResolver.ResolveIdentity(IClientConfig clientConfig) 
```

Comment:
@normj Wanted to get your opinion on this: I had to update the identity resolver interfaces so that the auth resolver pipeline handler could pass a config to them. 

This worked, but I wonder if making them "aware" of SDK components is something we should avoid (or if it's not an issue for now since the interfaces are defined in `Core` anyway).